tutor/src/screens/assignment-edit/clone/editor.js

```diff
@@ -0,0 +1,162 @@
+      & div:last-child {
+        .btn:not(:last-child) {
+          background: ${colors.neutral.lite};
+          border: 1px solid ${colors.neutral.lite};
+        }
+      }
+    }
+  }
+
+  @media ${breakpoints.xlUp} {
+  .modal-dialog {
+    max-width: 65%;
+  }
+}
+`;
+
+const WrqNote = styled.div`
+  padding: 0 1rem;
+  ul {
+    padding: 0 1.5rem;
+    color: ${colors.neutral.dark};
+  }
+`;
+
+@observer
+class Editor extends React.Component {
+
+  static propTypes = {
+    ux: PropTypes.instanceOf(UX).isRequired,
+    showModal: PropTypes.bool.isRequired,
+    onHide: PropTypes.func.isRequired,
+  }
+
+  async saveDraft() {
+    const { ux: { saveAsDraft }, onHide } = this.props;
+    await saveAsDraft();
+    onHide();
+  }
+
+  async publish() {
+    const { ux: { savePlan }, onHide } = this.props;
+    await savePlan();
+    onHide();
+  }
+
+  async editDetails() {
+    const { ux: { saveAsDraft, navigateToStep } } = this.props;
+    await saveAsDraft();
+    navigateToStep('details');
+  }
+
+  onCancel() {
+    this.props.onHide();
+  }
+
+  render() {
+    const { ux, showModal, onHide } = this.props;
+    return (
+      <StyledTemplateModal
+        show={showModal}
+        backdrop="static"
+        onHide={onHide}
+        templateType={ux.plan.type}
+      >
+        <Modal.Header closeButton>
+          {`Add details to ${ux.plan.title}`}
+        </Modal.Header>
+        <Modal.Body>
+          <Formik
+            initialValues={ux.formValues} 
+            validateOnMount={true}
+          >
+            {ux.renderMiniCloneEditor}
+          </Formik>
+          {/* How information about WRM and grading template. */}
+          {
+            !ux.isCloneOldAssignment && 
+            <WrqNote>
+              <strong><label>Note</label></strong>
+              <ul>
+                <li>OpenStax Tutor Beta allows you to assign points to questions.</li>
+                <li>You can assign Written Response Questions (manually-graded) to students.</li>
+                <li>By default, MCQs are worth 1 point, and WRQs are worth 2 points.</li>
+                <li>To assign WRQs or to set points, click on Edit other details.</li>
+              </ul>
+            </WrqNote>
+          }
+        </Modal.Body>
+        <Modal.Footer>
+          <div>
+            <Button
+              variant="default"
+              onClick={onHide}
+              size="lg"
+            >
+            Cancel
+            </Button>
+          </div>
+          <div>
+            <Button
+              className="other-button"
+              size="lg"
+              onClick={() => this.saveDraft()}
+            >
+            Save as Draft
+            </Button>
+            <Button
+              className="other-button"
+              size="lg"
+              onClick={() => this.publish()}
+            >
+            Publish
+            </Button>
+            <Button
+              size="lg"
+              onClick={() => this.editDetails()}
```

Comment:
`onClick` callbacks like this could just be: `onClick={this.editDetails}` if it was `@action.bound`